src/frontend/js/modules/datetime/datetime.run.js

```diff
@@ -4,11 +4,17 @@ require('./datetime.service.js');
   'use strict';
 
   angular.module('esn.datetime')
+    .config(function(tmhDynamicLocaleProvider) {
+      tmhDynamicLocaleProvider.localeLocationPattern('https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.15/i18n/angular-locale_{{locale}}.js');
```

Comment:
Why are we getting this from a CDN? Isn't everything in `node_modules/angular-i18n`?

Reply:
we need a dynamic translation here , translate all days , months of datepicker , like this https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.15/i18n/angular-locale_vi.js

Reply:
Yes but I mean we already have those files in `node_modules/angular-i18n`, so we don't need to download them from a CDN.

Reply:
resolved!